index.html

```diff
@@ -0,0 +1,41 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title id="title">Dr. Norman Borlaug</title>
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+      </head>
+
+
+<body class="text-center">
+    <div id="main" >
+        <h1>Dr. Norman Borlaug</h1>
+        <p style=font-size:15px> The man who saved a billion lives
+            </p>
+     <div id="img-div">
+         <img  id="image"
+         src="https://c2.staticflickr.com/4/3689/10613180113_fdf7bcd316_b.jpg">
+         <p> 
+                Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+         </p>
+     </div>       
+     <div id ="tribute-info"></div>
```

Comment:
-[] [REQUIRED] Please make sure to have space-less equal signs. such as:
`<div class="striped">` instead of `<div class = "striped">`